src/app/components/filebrowsermvs/filebrowsermvs.component.html

```diff
@@ -30,6 +30,7 @@
         (clickEvent)="onNodeClick($event)"
         (dblClickEvent)="onNodeDblClick($event)" 
         [style]="style"
+        noResultMsg="No results"
```

Comment:
This should be in [ ]

Atm, it seems like for MVS it doesn't work, but that's probably because of no brackets, unless data isn't being set to null somewhere.

Reply:
Isn't [] used just to bind parent component variables to prop value? Here, it's a simple string not a variable.

Reply:
I've checked the [docs](https://angular.io/guide/template-syntax#one-time-string-initialization) to confirm if I'm using the correct syntax. Please, can you check it once?

Reply:
Hey @shivanshs9 I apologize for the slow-ish reply. So what I meant to say is that this is for filebrowsermvs not filebrowseruss, where MVS controls the "Dataset" portion of the Editor. So I think the way you used it in filebrowseruss was correct, but I don't see this no results message when we click on "Datasets" in the Editor, so you should remove this part from filebrowsermvs as it currently does nothing, or make it work for both of them (same way you made it work for filebrowseruss, but this time for *mvs.component.html and *mvs.component.ts)

Reply:
Oh, I think I see what you were trying to do. You were trying to set a custom variable in uss, and then use the default in-line string you made in mvs, as to not have a string variable in mvs. That's fine, this will work, but you also need to bind it in zlux-editor\webClient\node_modules\@zlux\file-explorer\src\app\components\zlux-file-explorer

![image](https://user-images.githubusercontent.com/20528015/76796780-8a3f7980-67a2-11ea-9290-04982b6358b2.png)

*Either way* these are both different ways of doing things and it's good to use one or the other. Maintaining consistency makes the code easier to read and less messy as time goes on

Reply:
Hey @DivergentEuropeans
Sorry, but I wasn't trying to bind the value for _noResultMsg_ in **zlux-file-explorer** component. Is there any particular reason to do that though?
My goal was to pass _noResultMsg_ prop to **tree-root** component. To that end, I used a member variable for **FileBrowserUSSComponent** since the string is dynamic depending on data. But for **FileBrowserMVSComponent**, the string is static so I didn't use a variable and hence that syntax.
As you can find in the screenshot I've attached, it works on my end (I changed the string for testing purposes):
![ZLUX Editor](https://user-images.githubusercontent.com/8217199/77015414-bfaeb780-699a-11ea-8ccc-00017e9105eb.png)